include/nbind/signature/BaseSignature.h

```diff
@@ -178,7 +178,10 @@ class TemplatedBaseSignature : public BaseSignature {
 
 		if(!arityIsValid(nanArgs)) {
 			// TODO: When function is overloaded, this test could be skipped...
-			Nan::ThrowError("Wrong number of arguments");
+
+			char buffer [40];
+			sprintf (buffer, "Wrong number of arguments, expected %d", (sizeof...(Args)));
```

Comment:
Ok, changed that. The `sprintf` was not convincing neither to me... I'm not so proficient in C++, sorry :sob: